src/config.ts

```diff
@@ -102,8 +102,8 @@ const config = {
       clientHost: '127.0.0.1' as Host,
       clientPort: 0 as Port,
       maxReadBufferBytes: 1_000_000_000, // About 1 GB
-      pingInterval: 1_000, // 1 second
-      pingTimeout: 10_000, // 10 seconds
+      pingIntervalTime: 1_000, // 1 second
+      pingTimeoutTime: 10_000, // 10 seconds
```

Comment:
Let's say that `networkConfig` is generic across transport options (websocket, quic).

While `rpcConfig` is specific to `src/rpc` domain.

Then `proxyConfig` gets removed once we have fully migrated.

Reply:
So the difference between having values here and the domain specific creators having default values, is that these defaults for the entire application, whereas the domains may have defaults specific to the domain.

The `PolykeyAgent` and `PolykeyClient` should correspondingly not have any default values.